README.md

```diff
@@ -82,33 +91,52 @@ roslaunch mqtt_client standalone.launch
 ```
 
 ```txt
-[ WARN] [1652888439.858857758]: Parameter 'broker/tls/enabled' not set, defaulting to '0'
-[ WARN] [1652888439.859706411]: Parameter 'client/id' not set, defaulting to ''
-[ WARN] [1652888439.859717540]: Client buffer can not be enabled when client ID is empty
-[ WARN] [1652888439.860144376]: Parameter 'client/clean_session' not set, defaulting to '1'
-[ WARN] [1652888439.860366209]: Parameter 'client/keep_alive_interval' not set, defaulting to '0.000000'
-[ WARN] [1652888439.860583442]: Parameter 'client/max_inflight' not set, defaulting to '65535'
-[ INFO] [1652888439.860924591]: Bridging ROS topic '/ping' to MQTT topic 'pingpong'
-[ INFO] [1652888439.860967600]: Bridging MQTT topic 'pingpong' to ROS topic '/pong'
-[ INFO] [1652888439.861800203]: Connecting to broker at 'tcp://localhost:1883' ...
-[ INFO] [1652888440.062255501]: Connected to broker at 'tcp://localhost:1883'
+[ WARN] [1665575657.358869079]: Parameter 'broker/tls/enabled' not set, defaulting to '0'
+[ WARN] [1665575657.359798329]: Parameter 'client/id' not set, defaulting to ''
+[ WARN] [1665575657.359810889]: Client buffer can not be enabled when client ID is empty
+[ WARN] [1665575657.360300703]: Parameter 'client/clean_session' not set, defaulting to '1'
+[ WARN] [1665575657.360576344]: Parameter 'client/keep_alive_interval' not set, defaulting to '60.000000'
+[ WARN] [1665575657.360847295]: Parameter 'client/max_inflight' not set, defaulting to '65535'
+[ INFO] [1665575657.361281461]: Bridging ROS topic '/ping/ros' to MQTT topic 'pingpong/ros' 
+[ INFO] [1665575657.361303380]: Bridging primitive ROS topic '/ping/primitive' to MQTT topic 'pingpong/primitive' 
+[ INFO] [1665575657.361352809]: Bridging MQTT topic 'pingpong/ros' to ROS topic '/pong/ros'
+[ INFO] [1665575657.361370558]: Bridging MQTT topic 'pingpong/primitive' to primitive ROS topic '/pong/primitive'
+[ INFO] [1665575657.362153083]: Connecting to broker at 'tcp://localhost:1883' ...
+[ INFO] [1665575657.462622065]: Connected to broker at 'tcp://localhost:1883'
 ```
 
 Note that the *mqtt_client* successfully connected to the broker and also echoed which ROS/MQTT topics are being bridged.
 
-In order to test the communication, publish any message on ROS topic `/ping` and wait for a response on ROS topic `/pong`. To this end, open two new terminals and execute the following commands.
+In order to test the communication among *mqtt_clients*, publish any ROS message on ROS topic `/ping/ros` and wait for a response on ROS topic `/pong/ros`.
 
 ```bash
-# 1st terminal: listen on /pong
-rostopic echo /pong
+# 1st terminal: listen for ROS messages on /pong/ros
```

Comment:
The numbering of terminals in the quick start might be misleading. 

`docker run --rm --network host --name mosquitto eclipse-mosquitto` is the first terminal in the quick start.

`roslaunch mqtt_client standalone.launch` is the second terminal in the quick start.

So here, we would start our 3rd terminal and the next terminal numbers would need to be increased as well.

Reply:
Added sentence to open 5 new terminals, to which these numbers are then referring to.